lib/cc/analyzer/issue_path_is_file_validation.rb

```diff
@@ -0,0 +1,19 @@
+module CC
+  module Analyzer
+    class IssuePathIsFileValidation < Validation
+      def valid?
+        File.file?(path)
```

Comment:
I believe this will also return false for symlinks. I think that's fine but wanted to be sure that's ok with you